release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -40,6 +40,43 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-7-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-7-enhanced-disk-provisioning"]
+==== Enhanced disk provisioning for LUKS and RAID devices
+
+Disk tooling enhancements now support complex root devices in bare metal deployments. For example, software RAID improvements now allow for more robust nodes that can still reboot in the event of a drive failure. And support for multiple LUKS root devices means that you now have added capabilities such as using the Local Storage Operator in file mode, having your non-rootfs data encrypted at rest, and use of third-party software that requires local persistent storage.
+
+{op-system} 4.7 includes several improvements to disk provisioning for bare metal deployments:
+
+* Native Ignition support for LUKS disk encryption provides additional configurability for encrypted root filesystems, as well as support for encryption of additional data filesystems.
+* {op-system} now supports boot disk mirroring, except on s390x, providing redundancy in the case of disk failure.
+* {op-system} on s390x can be installed onto FBA DASD disks.
+
+[NOTE]
+====
+LUKS configuration can no longer be performed using `/etc/clevis.json`. Provisioning will fail if `/etc/clevis.json` is specified in the machine config.
+====
+
+[id="ocp-4-7-bootupd"]
+==== Update the bootloader using bootupd
+
+With bootupd, {op-system} users now have access to a cross-distribution, system-agnostic OS update tool that manages firmware and boot updates in UEFI and legacy BIOS boot modes that run on modern architectures.
```

Comment:
```suggestion
With `bootupd`, {op-system} users now have access to a cross-distribution, system-agnostic OS update tool that manages firmware and boot updates in UEFI and legacy BIOS boot modes that run on modern architectures.
```

This looks like a [particular system item](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference), so it should be enclosed in backticks.

Reply:
✅